code/modules/antagonists/gang/gang_themes.dm

```diff
@@ -0,0 +1,276 @@
+
+		/datum/antagonist/gang/henchmen = "HENCHMEN! It is me, your boss, <b>THE MONARCH!</b> I have sent you to this pitiful station with one goal, and one goal only! \
+		<B>MENACE THE RESEARCH DEPARTMENT!!!</B> \
+		The Research Director who is supposedly assigned to this station used to be friends with Doctor Venture back in college, and therefore HE MUST PAY!!! \
+		Keep those damned eggheads in the R&D department on their toes, and MENACE THEM!!! Commit dastardly villainous acts! <B>GO FORTH, HENCHMEN!</B>",
+
+		/datum/antagonist/gang/osi = "Greetings, agent. Your mission today is simple; \
+		The research department on board this station is about to be the target of a Level 10 Arching operation directed by The Monarch, a member of the Guild of Calamitious Intent. \
+		Protect and secure the Research Department with your life, but <B>do NOT allow them to complete their research.</B> Impede them in as many ways as possible without getting caught. \
+		If you encounter any of the Monarch's henchmen, make sure to obey Equally Matched Aggression levels, or you will be penalized by the top brass. \
+		Above all else, <B>Remain undercover as much as possible.</B> The station's crew CANNOT be allowed to know of our true nature, or we will see a repeat of the Second American Civil War. <br>\
+		The invisible one."
+	)
+
+/datum/gang_theme/real_smt_game
+	name = "Deciding The REAL Shin Megami Tensei Game"
+	description = "Wazzap, GAMERS! It's your boy, XxXx_360_NoScope_AnimeGamer_xXxX coming at you LIVE from 42.0! Tonight's argument: What makes a REAL Shin Megami Tensei game? \
+	Our guests tonight will settle this debate once and for all! <br>\
+	From the Traditional camp with the position 'only MAIN SMT games count', we've got a representative from the Jack Bros! <br>\
+	And from the new Radical camp with the position 'all SMT franchise games count', we've got a representative from the Phantom Thieves of Hearts! <br>\
+	We'll be right back with the debate after this word from our sponsors!"
+	involved_gangs = list(/datum/antagonist/gang/jackbros, /datum/antagonist/gang/phantom)
+	gang_objectives = list(
+
+		/datum/antagonist/gang/jackbros = "He-hello, friend-hos! We've got a nice chilly station out in space tonight! \
+		You know what would be cool? If we could chill out with our friends in the new Shadow government you're going to establish! <br>\
+		<B>Get all the station heads on board with the hee-ho vibes, and if they won't join up, then replace 'em with fellow hee-hos!</B> \
+		You might have to hee-urt some hos this time, but that's what you need to do to make things work!",
+
+		/datum/antagonist/gang/phantom = "For real? We get to stop a shadow government on a space station? That's awesome, bro! <br>\
+		We're the Phantom Thieves of Hearts, and we're gonna make all these shitty Heads of Staff confess to their crimes! <br>\
+		<B>Steal the hearts of the shitty Heads of Staff on the station and make 'em confess their crimes publicly!</B>\
```

Comment:
```suggestion
		<B>Steal the hearts of the Heads of Staff on the station and make 'em confess their crimes publicly!</B>\
```
you already called them shitty earlier, there's no need to repeat the insult.

Reply:
the phantom thieves are immature, intentional